source/type/pb_converter.cpp

```diff
@@ -1111,11 +1075,55 @@ void PbCreator::Filler::fill_pb_object(const WayCoord* way_coord, pbnavitia::Add
     }
 }
 
+void PbCreator::Filler::fill_pb_object(const nt::Contributor* c, pbnavitia::FeedPublisher* fp){
+        fp->set_id(c->uri);
+        fp->set_name(c->name);
+        fp->set_license(c->license);
+}
+
+const std::string& PbCreator::register_section(pbnavitia::Journey* j, size_t section_idx) {
```

Comment:
It's not mandatory, but wouldn't it be better to have a custom wrapper for journeys in order to split a bit a code ?